proposals/interceptors-issues-2024-01.md

```diff
@@ -0,0 +1,147 @@
+# Open issues for interceptors
+
```

Comment:
Would add a blurb here about how this is about closing the gap on open issues that would get us from preview to productivity. Locking down the existing feature set vs. opening the door for new ones.